node/downloader.go

```diff
@@ -0,0 +1,31 @@
+package node
+
+import (
+	"github.com/nknorg/nkn/v2/chain/db"
+	"github.com/nknorg/nkn/v2/util/log"
+
+	"github.com/nknorg/nkn/v2/common"
+)
+
+type Downloader struct {
```

Comment:
Do we even need this file and `Downloader`?

Reply:
For providing new peers and new pivot blockhash if syncer needed

Reply:
Everything in this file can be compressed as
```
log.Info()
s := newStateSync(d, root, peers)
s.run()
```
in `StartFastSyncing()`